src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,33 @@
+.nav__list {
+  display: flex;
+
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+
+  margin-right: 20px;
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav__link {
+  font-weight: 500;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+
+  line-height: 60px;
+
+  &:hover {
+    color: #00acdc;
+
+    transition: 0.3s;
+  }
+}
```

Comment:
re-write it and use more SASS.   for example you can change all parts "nav..."  to &

Reply:
You're right, I rewrote it